server/events/instrumented_project_command_runner.go

```diff
@@ -53,6 +54,10 @@ func (p *InstrumentedProjectCommandRunner) Import(ctx command.ProjectContext) co
 	return RunAndEmitStats("import", ctx, p.projectCommandRunner.Import, p.scope)
 }
 
+func (p *InstrumentedProjectCommandRunner) StateRm(ctx command.ProjectContext) command.ProjectResult {
+	return RunAndEmitStats("state rm", ctx, p.projectCommandRunner.Import, p.scope)
```

Comment:
Is this right? or should it be passing in a state rm specific input in its 3rd argument?